src/sentry.ts

```diff
@@ -51,6 +51,14 @@ const deClutterConfig = {
 
 export const SentryRoute = Sentry.withSentryRouting(Route)
 
+const checkForBlockedUserAgents = () => {
```

Comment:
Funny! When I first reviewed this I was going to suggest doing an approach like this to add a list of agents we wanted blocked, but then I thought it was a bit overkill, so funny we arrived to this on our own

Reply:
I pulled it out cause we're using it in two locations now ... so I thought it best they stay in sync with one another